deploy/charts/harvester/charts/cdi/README.md

```diff
@@ -0,0 +1,59 @@
+# KubeVirt Containerized Data Importer Helm Chart
+
+[`CDI`](https://github.com/kubevirt/containerized-data-importer/blob/dbab72c93e05957fe8eb5d8d861843d79f7998bf/pkg/apis/core/v1beta1/types.go#L230-L245) declaratively defines a desired [KubeVirt Containerized Data Importer](https://github.com/kubevirt/containerized-data-importer) setup to run in a Kubernetes cluster. It provides options to configure KubeVirt Containerized Data Importer bundle components to deploy.
+
+The current configuration management of KubeVirt Containerized Data Importer Operator for `CDI` resource has some limitations as below:
+
+- `CDI` resource is a cluster scope resource.
+- Only the first deployed one in multiple `CDI` resources is valid.
+
+## Chart Details
+
+This chart will do the following:
+
+- Deploy a `CDI` CRD resource named `cdi`.
+
+### Prerequisites
+
+- Kubernetes 1.16+ with Beta APIs enabled.
+- Helm 3.2+.
+- KubeVirt Containerized Data Importer Operator v1.21.0+ with Beta APIs enabled.
+
+### Installing the Chart
+
+To install the chart with the release name `my-release`.
+
+```bash
+$ # install chart to target namespace installed cdi-operator, i.e: cdi-system
+$ helm install my-release cdi --namespace cdi-system
```

Comment:
I don't know whether the user will want to install the CDI chart independently, but I would prefer to use `harvester-system` for better consistency.

Reply:
Updated